src/main/java/org/spongepowered/api/world/gen/PopulatorObject.java

```diff
@@ -28,7 +28,10 @@
 import org.spongepowered.api.world.World;
 
 /**
- * Represents a structure which can be placed into the world by a populator.
+ * Represents a relatively small object (must fit in a single chunk) that can be
```

Comment:
While it is true that the pieces that make up structures are typically less than a chunk in size it is by no means a requirement, in theory there is no limit to the size of a populator object. Especially since the objects placed by populators will often cross the boundary between two chunks which is why they are only placed in after the chunks in a 1 chunk radius around the target chunk have been generated.